docs/reference/method/MongoDBClient-listDatabases.txt

```diff
@@ -13,28 +13,27 @@ MongoDB\\Client::listDatabases()
 Definition
 ----------
 
-.. phpmethod:: MongoDB\\Client::listDatabases($options)
+.. phpmethod:: MongoDB\\Client::listDatabases()
 
-   Lists the available databases.
+   Returns information for all databases on the server.
 
    .. code-block:: php
 
       function listDatabases(array $options = []): MongoDB\Model\DatabaseInfoIterator
 
-   :phpmethod:`MongoDB\\Client::listDatabases` has the following parameters:
+   :phpmethod:`MongoDB\\Client::listDatabases()` has the following parameters:
```

Comment:
Is there value in using a `:phpmethod:` reference here instead of saying "This method"? I noticed that it only renders as fixed-width text instead of a link to the current page. Considering that we already repeat the method name twice immediately before this, I'm dubious that this is any better than common text like I'm using below for `$options`.